### PR TITLE
Bug 1213885 - Editing a pinned site opens a window called 'Edit bookmark'

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=59c8eda149b8660d7115b46e9513146d71439d00

File: apps/bookmark/js/activity_handler.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ var ActivityHandler = {
           window.removeEventListener('status-hidden', hidden);
           activity.postResult(saved ? 'saved' : 'updated');
         });
-        
+
         utils.status.show(
           navigator.mozL10n.get(saved ? 'added-to-home-screen-message' :
```
Comment (albertopq) [inline apps/bookmark/js/activity_handler.js]: I know is not from this PR, but can we use .formatValue instead? I think we are trying to remove .get

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=049c95bc53fe8644e5c3bef72417865d11e2a8b4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dd0ff39af01207cc46db2bc52ce85b615d2da9c4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ba680fa6d176473ca91e83717de37d26a1b1ca24

**Pull request merged**